sdk/storage/CMakeLists.txt

```diff
@@ -5,7 +5,8 @@ cmake_minimum_required (VERSION 3.12)
 
 project (azure-storage LANGUAGES CXX)
 set(CMAKE_CXX_STANDARD 14)
-set(CMAKE_CXX_STANDARD_REQUIRED True)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

Comment:
Should we guard it with `if(WIN32)`?

Reply:
I don't think so. it has not effect on Linux

Reply:
The variable name already implies it only works to Windows.